pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1401,7 +1401,7 @@ class Timedelta(_Timedelta):
     # Arithmetic Methods
     # TODO: Can some of these be defined in the cython class?
 
-    __inv__ = _op_unary_method(lambda x: -x, '__inv__')
+    __inverse__ = _op_unary_method(lambda x: -x, '__inverse__')
```

Comment:
shouldn't this be ```__invert__```?

Reply:
yep, will update

Reply:
huh looks like when i make this `__invert__` we get nutty results instead of raising.  better to remove it entirely and keep the test that we raise

Reply:
updated + greenish (windows timeout)